### PR TITLE
User and Product Devlopment

### PR DESCRIPTION
**Pull request by Harsh-B-Patel**

### CODE REVIEW

**Comment by yumstar** [comment]
I would recommend that you rename the project path to just `src/ecommerce` if possible.

File: CreateSampelTables.sql
Comment (yumstar) [inline CreateSampelTables.sql:1]: `CreateSampleTables.sql`?

Comment (Harsh-B-Patel) [reply CreateSampelTables.sql:1]: Just some data, to add in My SQL. I will move it down to resources folder

**Comment by Harsh-B-Patel** [comment]
src/main/java/ is industry wide standard. Even though a smaller path would be nice. This is used industry wide.

**Pull request merged**